SchoolWeek.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <iostream>
+#include "LeakedObjectDetector.h"
+#include "SchoolDay.h"
+
+struct SchoolWeek 
+{
+    SchoolWeek();
+    ~SchoolWeek();
+    SchoolDay monday;
+    SchoolDay tuesday;
+    SchoolDay wednesday;
+    SchoolDay thursday;
+    SchoolDay friday;
+    
+    void makeHalfDay( SchoolDay day, float lengthDecrease );
+    void snowDay( SchoolDay day);
```

Comment:
From Project 5 Part 3:
`7) If you were using any UDTs as function arguments like this:`
```
void someMemberFunction(Axe axe);
```
`Pass those arguments by Reference now that you know what references are (Project 6 Part 2)`
```
void someMemberFunction(Axe& axe);
```
`If you aren't modifying the passed-in object inside the function, pass by const reference`
```
void someMemberFunction(const Axe& axe);
```
- [ ] revise